parts/k8s/addons/kubernetesmasteraddons-keyvault-flexvolume-installer.yaml

```diff
@@ -47,3 +47,87 @@ spec:
         name: volplugins
       nodeSelector:
         beta.kubernetes.io/os: linux
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    app: blobfuse-flexvolume
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: "EnsureExists"
+  name: blobfuse-flexvolume
+  namespace: kv
+spec:
+  template:
+    metadata:
+      labels:
+        app: blobfuse-flexvolume
+        kubernetes.io/cluster-service: "true"
+        addonmanager.kubernetes.io/mode: "EnsureExists"
+    spec:
+      tolerations:
+      containers:
+      - name: blobfuse-flexvolume
+        image: "ritazh/kv-flexvol-installer:v0.0.3"
```

Comment:
we'll wanna put the right image references here (also maybe file a backlog issue to publish them somewhere more general)

Reply:
blobfuse: https://github.com/Azure/kubernetes-volume-drivers/blob/master/flexvolume/blobfuse/deployment/blobfuse-flexvol-installer-1.8.yaml

smb: https://github.com/Azure/kubernetes-volume-drivers/blob/master/flexvolume/smb/deployment/smb-flexvol-installer.yaml

Reply:
> publish them somewhere more general

yes agreed. once I get ritazh/kv-flexvol-installer moved over to mcr.microsoft.com/k8s/, will open another PR to update this.